### PR TITLE
fix : Added missing inputType attribute to EditTexts

### LINKED ISSUE #473: Missing inputType attribute for some EditTexts
**Summary:**

There are some EditTexts for which inputType attribute is missing. The inspection gives the following result.

Screenshot from 2019-03-11 23-36-09

I am going to add missing inputType attributes to those EditTexts.

### PR DESCRIPTION
**Pull request by RajaVamsi11**
Fixes #473 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Roshan13046** [comment]
Where are the project maintainers?
There are many issues which need to be closed...

**Comment by Prashant830** [comment]
@RajaVamsi11  I think, In your branch have a code conflict . Are u still able to resolved it??

File: mifospay/src/main/res/layout/fragment_kyc_lvl3.xml [hunk 1]
```diff
@@ -19,6 +19,7 @@
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:hint="@string/pan_id"
+            android:inputType="text"
```
Comment (rchtgpt) [inline mifospay/src/main/res/layout/fragment_kyc_lvl3.xml:22]: PAN ID must be an alphanumeric keyboard with only uppercase characters.

File: mifospay/src/main/res/layout/fragment_transfer.xml [hunk 2]
```diff
@@ -91,6 +91,7 @@
                           android:layout_width="match_parent"
                           android:layout_height="wrap_content"
                           android:hint="@string/virtual_payment_address"
+                          android:inputType="textEmailAddress"
```
Comment (rchtgpt) [inline mifospay/src/main/res/layout/fragment_transfer.xml:94]: Based on my understanding, a VPA looks more like a username than an email address. So, this inputType wouldn't really help our case. We should instead set the `inputType` to `text`.

**Comment by rchtgpt** [comment]
Thanks for bringing up the issue, @RajaVamsi11! We're gonna have to close this PR due to inactivity and merge conflicts but we're working to resolve the issue.

**Pull request closed without merging**